### PR TITLE
fix: Fix move the filedrop module to the commons extensions - EXO-68825

### PR DESCRIPTION
**Pull request by sofyenne**
Prior to this change, after moving the filedrop module from the commons to the perk-store addon, by uninstalling the perk-store addon from the exo server, we were not able to add an illustration to the news or drop a file on the chat application. This change is going to add the filedrop module to the commons extensions resources to resolve this issue.

### CODE REVIEW

File: commons-extension-webapp/src/main/webapp/WEB-INF/gatein-resources.xml [hunk 1]
```diff
@@ -36,5 +36,14 @@
...
   </module>
+  <module>
+    <name>filedrop</name>
+    <script>
+      <path>/javascript/lib/jquery.filedrop.js</path>
+    </script>
+    <depends>
+      <module>jquery</module>
+    </depends>
+  </module>
```
Comment (boubaker) [inline commons-extension-webapp/src/main/webapp/WEB-INF/gatein-resources.xml:47]: Please move it to social, all libs are there now, except for CKEditor...

**Review by boubaker — CHANGES_REQUESTED** [review]

**Review by mkrout — APPROVED** [review]

**Review by boubaker — CHANGES_REQUESTED** [review]
BTW, this library is used by Perk Store only in context of meeds, it should remain there since we aim to delete it in favor of new way introduced here by @SaraBoutej :
https://github.com/Meeds-io/social/blob/611021e9b80df93619e09cb4ce9597ae946f572c/webapp/portlet/src/main/webapp/vue-apps/common/initComponents.js#L198

A `v-draggable` Vue directive.

**Comment by mkrout** [comment]
> BTW, this library is used by Perk Store only in context of meeds, it should remain there since we aim to delete it in favor of new way introduced here by @SaraBoutej : https://github.com/Meeds-io/social/blob/611021e9b80df93619e09cb4ce9597ae946f572c/webapp/portlet/src/main/webapp/vue-apps/common/initComponents.js#L198
> 
> A `v-draggable` Vue directive.

It's also needed for news and chat in exo context, so we need to keep it in the product side until the 'v-draggable' is used in all projects

**Comment by boubaker** [comment]
> so we need to keep it in the product side until the 'v-draggable' is used in all projects

@mkrout , you can redefine it in eXo context only by using a specific name, no need to exclusively use the name `filedrop`, it can be even `exofiledrop`. This library has been deprecated and isolated to be removed after rework of perkstore, thus no need to re-copy it into common or social module. You can operate the change into `commons-exo` module.

**Comment by mkrout** [comment]
> > so we need to keep it in the product side until the 'v-draggable' is used in all projects
> 
> @mkrout , you can redefine it in eXo context only by using a specific name, no need to exclusively use the name `filedrop`, it can be even `exofiledrop`. This library has been deprecated and isolated to be removed after rework of perkstore, thus no need to re-copy it into common or social module. You can operate the change into `commons-exo` module.

Ok sounds good for me

**Pull request closed without merging**